expression/aggregation/descriptor.go

```diff
@@ -110,37 +110,64 @@ func (a *AggFuncDesc) typeInfer(ctx sessionctx.Context) {
 	}
 }
 
-// CalculateDefaultValue gets the default value when the aggregation function's input is null.
+// EvalNullValueInOuterJoin gets the null value when the aggregation is upon an outer join,
+// and the aggregation function's input is null.
+// If there is no matching row for the inner table of an outer join,
+// an aggregation function only involves constant and/or columns belongs to the inner table
+// will be set to the null value.
 // The input stands for the schema of Aggregation's child. If the function can't produce a default value, the second
 // return value will be false.
-//
-// According to MySQL, DefaultValue of the aggregation function can be tested as the following sql:
-//
+// e.g.
+// mysql> create table t(a int);
```

Comment:
Can we simplify the comment? This format is not very formal 😂

Reply:
ok, I'll have a try